setup.py

```diff
@@ -3,6 +3,7 @@
 
 from __future__ import with_statement
 
+from io import open
```

Comment:
Out of curiosity - is this for Python 2 compatibility?

Reply:
Yes.